LISA/LISAImplementation.cpp

```diff
@@ -249,7 +250,7 @@ class LISAImplementation : public Exchange::ILISA {
         _notificationCallbacks.push_back(notification);
         notification->AddRef();
 
-        TRACE_GLOBAL(Trace::Information, (_T("Register INotification:%p"), notification));
+        INFO("Register INotification: ", notification);
```

Comment:
Is the pointer going to be correctly logged here?

Reply:
verified, works as expected